app/javascript/components/patient/Patient.js

```diff
@@ -206,8 +207,13 @@ class Patient extends React.Component {
               <Col className="text-truncate">
                 <span className="font-weight-normal">Phone:</span>{' '}
                 <span className="font-weight-light">
-                  {this.props.details.primary_telephone && `${this.formatPhoneNumber(this.props.details.primary_telephone)}`}
+                  {this.props.details.primary_telephone && `${this.formatPhoneNumber(this.props.details.primary_telephone)} `}
                 </span>
+                {this.props.details.blocked_sms && (
+                  <Form.Label className="nav-input-label text-danger">
```

Comment:
Do you need `Form.Label` for this? Might be doable adding styling to a `<span>`.